data/pages-content/photos.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Photos
+subtitle: Savour, relive, share the highlights of EuroPython 2022
+---
+
+# EuroPython 2022 Photos
+
+### Official Photos
```

Comment:
A one liner afterwards, something like: "Come check out the official photographs of the fun-filled Pythonic week we shared in Dublin and remote"

you're better at words than I am, so feel free to ignore this :p

Reply:
Added in https://github.com/EuroPython/website/pull/206/commits/0125ae4b59019fed983d71064e048a6722e3eee7

Reply:
Added in https://github.com/EuroPython/website/pull/206/commits/b0f09b824afafafbc2850ad2b03da3b8f6108e6b